apps/uvc-app/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 2.8)
+project(uvc-app)
```

Comment:
What is the reason for changing the uvc-gadget name to uvc-app?

Reply:
Named it wrong when I wrote the `CMakeLists.txt`. Should be fixed now.